content/doc/book/managing/system-properties.adoc

```diff
@@ -1601,6 +1601,17 @@ properties:
     To make Jenkins load time 8x faster (assuming sufficient IO), increase it to 8x.
     For example, 24 CPU Jenkins controller host use this: `-Djenkins.InitReactorRunner.concurrency=192`
 
+- name: jenkins.health.HealthCheckAction.thresholdTimeout
+  tags:
+  - tuning
+  def: |
+    `PT10S` (10 seconds)
+  since: <tbu>
+  description: |
+    Log a thread dump when a `/health` check exceeds this timeout to help diagnose stuck requests.
+    Useful for debugging Kubernetes liveness probe failures caused by stuck health checks.
+    The value must be in ISO-8601 duration format (e.g., `PT30S` for 30 seconds).
```

Comment:
> must be in ISO-8601 duration format...

Actually the https://javadoc.jenkins.io/jenkins/util/SystemProperties.html#getDuration(java.lang.String)
also accepts `30seconds` or `30s` as well.

Reply:
updated now.